specification/desktopvirtualization/DesktopVirtualization.Management/models.tsp

```diff
@@ -1442,12 +1611,51 @@ model DiskInfoProperties {
   /**
    * The disk type used by virtual machine in hostpool session host.
    */
+  @removed(Versions.v2025_04_01_preview)
   type: VirtualMachineDiskType;
+
+  /**
+   * The managed disk parameters.
+   */
+  @added(Versions.v2025_04_01_preview)
+  managedDisk?: ManagedDiskProperties;
+
+  /**
+   * Specifies the ephemeral Disk Settings for the operating system disk used by the virtual machine.
```

Comment:
> ephemeral Disk Settings [](http://example.com/codeflow?start=18&length=24)

nit, no need to capitalize ' ephemeral Disk Settings'
can just be ' ephemeral disk settings